src/include/btmem.h

```diff
@@ -128,6 +128,46 @@ __wt_page_header_byteswap(WT_PAGE_HEADER *dsk)
 #define WT_PAGE_HEADER_BYTE(btree, dsk) \
     ((void *)((uint8_t *)(dsk) + WT_PAGE_HEADER_BYTE_SIZE(btree)))
 
+/*
+ * WT_DELTA_HEADER --
+ *	Header for delta
+ */
+struct __wt_delta_header {
+    /*
+     * Memory size of the delta.
+     */
+    uint32_t mem_size; /* 0-03: in-memory size */
+
+    uint8_t type; /* 04: page type */
+
+    uint8_t flags; /* 05: flags */
+
+    /* A byte of padding, positioned to be added to the flags. */
+    uint8_t unused; /* 06: unused padding */
+
+    uint8_t version; /* 07: version */
+};
```

Comment:
It looks like we need to be careful about the size here, should we add one of those "struct size check" lines (or a TODO to add one?)

Reply:
I don't see we check for page header. Will leave it for now.